src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+
+import moment from '../../lib/moment-tz';
+
+import { FETCH_STATUS } from '../../utils/constants';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import * as actions from '../redux/actions';
+import AppointmentDateTime from './cards/confirmed/AppointmentDateTime';
+import { getVARFacilityId } from '../../services/appointment';
+import AppointmentInstructions from './cards/confirmed/AppointmentInstructions';
+import AddToCalendar from '../../components/AddToCalendar';
+import { selectFeatureCancel } from '../../redux/selectors';
+import FacilityAddress from '../../components/FacilityAddress';
+
+function CommunityCareAppointmentDetailsPage({
+  appointmentDetails,
+  appointmentDetailsStatus,
+  fetchConfirmedAppointmentDetails,
+  confirmedStatus,
+}) {
+  const { id } = useParams();
+  const history = useHistory();
+
+  const appointment = appointmentDetails?.[id];
+
+  useEffect(() => {
+    const status = confirmedStatus === FETCH_STATUS.succeeded;
+
+    if (!status) {
+      history.push('/');
+    }
+
+    if (!appointment) {
+      fetchConfirmedAppointmentDetails(id);
+    }
+
+    scrollAndFocus();
+  }, []);
+
+  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+    return (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your confirmed appointments..." />
+      </div>
+    );
+  }
+
+  if (!appointment) {
```

Comment:
I'll make a ticket for this, but I think we want to combine this no `appointment` state with the loading state above so that we render the loading indicator on the first render of this page. Otherwise, our `scrollAndFocus` call doesn't focus on anything after the first render and a screen reader user isn't notified about the loading state.

We'd need to fix this on several pages, so no need to fix it here yet.